src/dmd/backend/os.c

```diff
@@ -161,7 +169,7 @@ void *globalrealloc(void *oldp,size_t newsize)
  * Functions to manage allocating a single virtual address space.
  */
 
-void *vmem_reserve(void *ptr,unsigned long size)
+void *vmem_reserve(void *ptr,ulong size)
```

Comment:
This is not correct. A C `unsigned long` is a `uint` in D, not a `ulong`.

Reply:
`unsigned long` should be `core.stdc.config.c_ulong`. `long` in C is sometimes 32bit and sometimes 64bit, deepening on which platform. @thewilsonator ^

Reply:
this function and I think the vast majority of this file is completely unused.

Reply:
Hence why despite being wrong went through green. I plan to do a dead strip of DMD at some point.